targets/csharp/source/PlayFabSDK/source/Qos/QosResult.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace PlayFab.QoS
+{
+    using System.Collections.Generic;
+
+    public class QosResult
+    {
+        public List<QosRegionResult> RegionResults { get; set; }
+
+        public int ErrorCode { get; set; }
```

Comment:
This ErrorCode is always compared against an enum QosErrorCode value, cast to an integer. Why isn't this just a QosErrorCode?

Reply:
Creating an error message instead. 

---
In reply to: [288680173](https://github.com/PlayFab/SDKGenerator/pull/535#discussion_r288680173) [](ancestors = 288680173)